src/pivot-table/components/shared-styles.ts

```diff
@@ -90,3 +91,27 @@ export const getTotalCellDividerStyle = ({
 
   return style;
 };
+
+interface StyledLeftGridProps {
+  leftGridWidth: number;
+}
+
+export const StyledLeftGrid = styled("div", {
+  shouldForwardProp: (prop: string) => !["leftGridWidth"].includes(prop),
+})(({ leftGridWidth }: StyledLeftGridProps) => ({
+  width: leftGridWidth,
```

Comment:
Not sure this is a great idea. I think that every time the chart is re-size or a column is re-size a new css class will be created by mui